io_scene_niftools/operators/common_op.py

```diff
@@ -89,17 +89,17 @@ class CommonImportScale:
     scale_correction_import: bpy.props.FloatProperty(
         name="Scale Correction Import",
         description="Changes size of mesh to fit onto Blender's default grid.",
-        default=1.0,
-        min=0.01, max=100.0, precision=2)
+        default=0.1,
```

Comment:
Can we please use the same factor everywhere and call it "Scale Correction"?
Apply x on import and 1/x on export, or the other way around.
Having two values is counter-intuitive.

Reply:
Indeed, based on our most recent discussion, I had made the assumption that we should have individual values.
As you said the need to keep scaling consistent across the operators would be best suited via a "global"
I think we can still have the control in the UI, but just driven by a global value.

Realisitically I won't have any time soon to do development work, would you be open to merging this and contributing this as a follow on?

Reply:
Yeah I should be able to do that.

Reply:
Happy to merge then with improvement required to finalise the design

Reply:
Sounds good, ready to merge then